ui/shared/components/panel/search/VariantSearchFormPanels.jsx

```diff
@@ -186,9 +186,15 @@ const PanelContent = React.memo(({ name, fields, fieldProps, helpText, fieldLayo
       {helpText && <i>{helpText} <VerticalSpacer height={20} /></i>}
       <Form.Group widths="equal">
         <Form.Field width={2} />
-        {fieldLayout ? fieldLayout(fieldComponents) : fieldComponents}
+        {fieldLayout ? fieldLayout(fieldComponents) : fieldComponents.slice(0, 6)}
         <Form.Field width={2} />
       </Form.Group>
+      {fieldComponents.length > 6 && !fieldLayout &&
```

Comment:
don't modify the default behavior here. Instead, pass `fieldLayout` to the frequency field and explicitly style the layout. `fieldLayout` if a function that will be called with list of `fieldComponents` and then returns the layout you want to render

Reply:
New layout:
![image](https://user-images.githubusercontent.com/33550059/118871942-0f965080-b8b6-11eb-8a4d-46d260121846.png)